src/App.tsx

```diff
@@ -11,35 +11,48 @@ import { getTodos } from './api';
 import { TodoModal } from './components/TodoModal';
 
 export const App: React.FC = () => {
+  const Values = {
+    All: 'all',
+    Active: 'active',
+    Completed: 'completed',
+  };
```

Comment:
use enum - not such constants

Reply:
ask in fe_chat if you need help

Reply:
Thank you, looks like I found where was an issue.